src/viam/rpc/dial.py

```diff
@@ -160,8 +162,11 @@ class _Runtime:
     _lib: ctypes.CDLL
     _ptr: ctypes.c_void_p
     _semaphore: PointerCounter = PointerCounter()
+    _lock: threading.Lock = threading.Lock()
```

Comment:
Since we are using `_semaphore` and `_lock` as [class variables now](https://github.com/viamrobotics/viam-python-sdk/pull/162/files#diff-e4e6e5a5f62866d78507a238287a5c75651040234a5d11e6ab0ee089acbca90dR168-R169), these should now be 

```python3
from typing import ClassVar

_semaphore: ClassVar[PointerCounter] = PointerCounter()
_lock: ClassVar[threading.Lock] = threading.Lock()
```

(Currently they're instance variables with default values)